README.md

```diff
@@ -10,6 +10,7 @@ The following folders can be found in this repository:
 - **[`iris`](https://archive.ics.uci.edu/ml/datasets/iris)** - The iris flower data set from Ronald Fisher's 1936 paper
 - **[`metriccentury`](https://github.com/mikeblas/samples-junk/tree/main/metriccentury)** - Data recorded from a 100 km bike ride
 - **[`DeNiro`](https://people.sc.fsu.edu/~jburkardt/data/csv/csv.html)** - Data on Robert De Niro's movies up to 2016
+- **`parquet`** - Sample parquet data in many formats
```

Comment:
I think this organization is not consistent with the existing repo.  In the existing repo, data is divided up into logical data groups.  Within those groups, there may be csv, parquet, tsv, etc. files.  The current parquet directory is not one data set.  It is at least (1) a taxi data set and (2) files for testing every combination and permutation of parquet.  The former should be appropriately separated.  I'm not convinced that the latter belongs here.  Do we really want every user that needs example data pulling all of these files?  The files may be small, so that it is ok.... but the users should understand what the data is, so that they don't waste time looking at it or trying to use it.

Reply:
added parquet directory with commit 21be2ca